corehq/apps/app_manager/templates/app_manager/partials/releases/build_profiles.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 {% load hq_shared_tags %}
+<a href="{% url 'manage_releases_by_app_profile' domain %}">Manage Releases By App Profile</a>
```

Comment:
this is just adding the link, we would also need to check if we need to show it or not based on permissions.

Reply:
Link text needs to go through `trans`, please.